xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecDRMPRIME.cpp

```diff
@@ -96,8 +96,30 @@ std::unique_ptr<CDVDVideoCodec> CDVDVideoCodecDRMPRIME::Create(CProcessInfo& pro
   return nullptr;
 }
 
+static bool SupportsDRM()
+{
+  bool hasDRM = false;
+  AVHWDeviceType type = AV_HWDEVICE_TYPE_NONE;
+  while((type = av_hwdevice_iterate_types(type)) != AV_HWDEVICE_TYPE_NONE)
+  {
+    if (type == AV_HWDEVICE_TYPE_DRM)
+    {
+      hasDRM = true;
+      break;
+    }
+  }
+
+  return hasDRM;
+}
+
 void CDVDVideoCodecDRMPRIME::Register()
 {
+  if (!SupportsDRM())
+  {
+    CLog::Log(LOGERROR, "FFMPEG has no DRM support");
```

Comment:
Probably not an error as this can be an expected path. Maybe INFO or WARNING is better. Also might want to include the class name here.